docs/function.md

```diff
@@ -110,231 +108,185 @@ let eleven = addFive(6);
 let twelve = addFive(7);
 ```
 
-Actually, the above `add` is nothing but syntactic sugar for this:
+If you want to partially call a function with an argument that is not the first
+argument use a `_` to skip arguments:
 
 ```reason
-let add = (x) => (y) => x + y;
+let divide = (a, b) => a / b;
+let half = divide(_, 2);
+let five = half(10);
 ```
 
-OCaml optimizes this to [avoid the unnecessary function allocation](https://reasonml.github.io/en/try.html?reason=DYUwLgBAhgJjEF4IAoAeBKRA+FBPTCOqEA1BLgNwBQVA9AFQTAD2zA1tJGABYgTMBXMAAchAQmhwAYgEsAbnxkBnaBAD6SmQDsA5qDUQAZgK0BjMDOZaIpqMGAT6tKqEiwYshYkkxkAVnRqF3AITWIkd08QZABGQKA) (2 functions here, naively speaking) whenever it can! This way, we get
+### Non-Mandatory Arguments
+
+When defining a function, named arguments can be marked as non-mandatory.
```

Comment:
Looks great. Not a mention of "option" anywhere (confusing).
In future iterations we could consider the term "opt-in arguments".